scripts/core/kernel.yml

```diff
@@ -158,12 +160,15 @@ params:
     - type: $x_kernel_group_info_t
       name: propName
       desc: "[in] name of the work Group property to query"
-    - type: "size_t"
+    - type: "uint32_t"
       name: propSize
       desc: "[in] size of the Kernel Work Group property value"
     - type: "void*"
       name: propValue
-      desc: "[in,out][range(0, propSize)] value of the Kernel Work Group property."
+      desc: "[in,out][optional][range(0, propSize)] value of the Kernel Work Group property."
+    - type: "uint32_t*"
```

Comment:
Same here, should these be `uint_32_t` or `size_t`?

Reply:
👍